src/libstd/net/tcp.rs

```diff
@@ -154,8 +164,7 @@ impl TcpStream {
     /// # Note
     ///
     /// Some platforms do not provide access to the current timeout.
-    #[unstable(feature = "socket_timeout", reason = "RFC 1047 - recently added",
-               issue = "27773")]
+    #[stable(feature = "socket_timeout", since = "1.4.0")]
```

Comment:
Huh... https://msdn.microsoft.com/en-us/library/windows/desktop/ms738544%28v=vs.85%29.aspx says it doesn't work.  But if the test works, then it must work, I guess.  Weird.